cli/admin/src/main/java/io/pravega/cli/admin/AdminCommandState.java

```diff
@@ -47,7 +51,14 @@
     public AdminCommandState() throws IOException {
         this.configBuilder = ServiceBuilderConfig.builder();
         try {
-            this.configBuilder.include(System.getProperty(ServiceBuilderConfig.CONFIG_FILE_PROPERTY_NAME, "conf/admin-cli.properties"));
+            Properties properties = new Properties();
+            try (FileReader reader = new FileReader(System.getProperty(ServiceBuilderConfig.CONFIG_FILE_PROPERTY_NAME, "conf/admin-cli.properties"))) {
```

Comment:
Is `admin-cli.properties` a generated file? I don't see any changes to this file that replaces the existing values with Env variable names.

Reply:
This will be handled in #6506.